CONTRIBUTING.md

```diff
@@ -192,17 +199,18 @@ Note: These instructions assume the user is using `mamba` instead of `conda` sin
    Once Morpheus has been built, it can be installed into the current virtual environment.
 1. Test the build (Note: some tests will be skipped)
    ```bash
-   pytest
+   pytest ${MORPHEUS_ROOT}/tests
```

Comment:
Whats the reasoning for specifying `${MORPHEUS_ROOT}/tests`? This should work out of the box

Reply:
Mostly to avoid any potentially confusing behavior, in my environment i have a ./scratch directory that just has other files/libs things I'm experimenting with, some of which have python files, and if you run `pytest` and not `pytest ./tests`, you'll see something like this:

```
platform linux -- Python 3.8.13, pytest-7.1.2, pluggy-1.0.0
benchmark: 3.4.1 (defaults: timer=time.perf_counter disable_gc=False min_rounds=5 min_time=0.000005 max_time=1.0 calibration_precision=10 warmup=False warmup_iterations=100000)
rootdir: /home/drobison/Development/devin-morpheus-public, configfile: pyproject.toml
plugins: benchmark-3.4.1, Faker-12.3.0, cov-3.0.0
collecting 23467 items / 46 errors / 8 skipped  
````

Reply:
Ok that makes sense. Could pick up other pytest files in things like `.cache`. Looks like we could add `./tests` to the pyproject.toml configuration file which would make this setting permanent: https://docs.pytest.org/en/6.2.x/customize.html#pyproject-toml. Thoughts about that?